example/kotlinlib/android/1-hello-world/app/src/main/java/com/helloworld/app/MainActivity.kt

```diff
@@ -23,10 +23,13 @@ class MainActivity : Activity() {
         textView.gravity = Gravity.CENTER
 
         // Set layout parameters (width and height)
-        textView.layoutParams = LayoutParams(
-            LayoutParams.MATCH_PARENT,
-            LayoutParams.MATCH_PARENT
-        )
+        textView.layoutParams = LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT)
+
+        // Set the text color using a resource
+        textView.setTextColor(getColor(R.color.text_green)) // Using hex color code directly
```

Comment:
What does `// Using hex color code directly` mean?

Reply:
😂😂😂 actually these documentations are not updated i will update them with updation of build docs